library/lib/core/application.cpp

```diff
@@ -435,6 +433,27 @@ void Application::toggleFramerateDisplay()
     // To be implemented (call setDisplayFramerate)
 }
 
+void Application::setGlobalQuit(bool enabled) {
+    Application::globalQuitEnabled = enabled;
+    for (auto it = Application::activitiesStack.begin(); it != Application::activitiesStack.end(); ++it) {
+        if (enabled)
+            Application::gloablQuitIdentifier = (*it)->registerExitAction();
+        else
+            (*it)->unregisterAction(Application::gloablQuitIdentifier);
+    }
+}
+
+void Application::setGlobalFPSToggle(bool enabled) {
+    Application::globalFPSToggleEnabled = enabled;
+    for (auto it = Application::activitiesStack.begin(); it != Application::activitiesStack.end(); ++it) {
+        if (enabled)
+            Application::gloablFPSToggleIdentifier = (*it)->registerAction(
+                "FPS", BUTTON_BACK, [](View* view) { Application::toggleFramerateDisplay(); return true; }, true);
```

Comment:
It looks like I missed this, sorry, the FPS toggle should also be in its own (private?) method to avoid redudancy between that and line 565

Reply:
Done. :+1: